### ISSUE TITLE
NetworkXNotImplemented: not implemented for multigraph type

### ISSUE THREAD

**Issue by kzhanar** [gboeing/osmnx#178]
Notebook 00-osmnx-features-demo
```
# edge closeness centrality: convert graph to line graph so edges become nodes and vice versa
edge_centrality = nx.closeness_centrality(nx.line_graph(G))

NetworkXNotImplemented                    Traceback (most recent call last)
<ipython-input-8-6ee8e3311955> in <module>()
      1 # edge closeness centrality: convert graph to line graph so edges become nodes and vice versa
----> 2 edge_centrality = nx.closeness_centrality(nx.line_graph(G))

<decorator-gen-502> in line_graph(G, create_using)

/opt/conda/lib/python3.6/site-packages/networkx/utils/decorators.py in _not_implemented_for(not_implement_for_func, *args, **kwargs)
     69         if match:
     70             msg = 'not implemented for %s type' % ' '.join(graph_types)
---> 71             raise nx.NetworkXNotImplemented(msg)
     72         else:
     73             return not_implement_for_func(*args, **kwargs)

NetworkXNotImplemented: not implemented for multigraph type

```
When running # edge closeness centrality: convert graph to line graph so edges become nodes and vice versa
edge_centrality = nx.closeness_centrality(nx.line_graph(G)) 
Error   NetworkXNotImplemented: not implemented for multigraph type

Version '0.8.1'

**Comment by gboeing** [comment]
See #157 and https://github.com/gboeing/osmnx-examples/issues/8

**Comment by Tahseen90** [comment]
import osmnx as ox 

name_place = "Kamppi, Helsinki, Finland"
graph = ox.graph_from_place(name_place)
graph_proj = ox.project_graph(graph)
fig, ax = ox.plot_graph(graph_proj)
edges = ox.graph_to_gdfs(graph_proj,nodes=False, edges=True)
edges.plot()

area = edges.unary_union.convex_hull.area

stats = ox.basic_stats(graph_proj, area==area)
extended_stats = ox.extended_stats(graph_proj, ecc=False, bc=True, cc=True)

when running the program ...

Traceback (most recent call last):

  File "<ipython-input-48-37bfef789ee5>", line 1, in <module>
    extended_stats = ox.extended_stats(graph_proj, ecc=False, bc=True, cc=True)

  File "/Users/mac/opt/anaconda3/envs/Tahseen3.6/lib/python3.6/site-packages/osmnx/stats.py", line 422, in extended_stats
    betweenness_centrality = nx.betweenness_centrality(G, weight='length')

  File "</Users/mac/opt/anaconda3/envs/Tahseen3.6/lib/python3.6/site-packages/decorator.py:decorator-gen-251>", line 2, in betweenness_centrality

  File "/Users/mac/opt/anaconda3/envs/Tahseen3.6/lib/python3.6/site-packages/networkx/utils/decorators.py", line 464, in _random_state
    return func(*new_args, **kwargs)

  File "</Users/mac/opt/anaconda3/envs/Tahseen3.6/lib/python3.6/site-packages/decorator.py:decorator-gen-250>", line 2, in betweenness_centrality

  File "/Users/mac/opt/anaconda3/envs/Tahseen3.6/lib/python3.6/site-packages/networkx/utils/decorators.py", line 80, in _not_implemented_for
    raise nx.NetworkXNotImplemented(msg)

NetworkXNotImplemented: not implemented for multigraph type

**Comment by gboeing** [comment]
Fixed in #350
